go.mod

```diff
@@ -4,8 +4,8 @@ go 1.15
 
 require (
 	github.com/ONSdigital/dp-healthcheck v1.0.5
-	github.com/ONSdigital/dp-net v1.0.9 // indirect
 	github.com/ONSdigital/log.go v1.0.1
+	github.com/ONSdigital/log.go/v2 v2.0.5 // indirect
```

Comment:
Same question here - is v1 still needed? If so it be good to know why its still required.

Reply:
as in https://github.com/ONSdigital/dp-authorisation/pull/13 - rookie mistake I think. I'll remove the references to v1 here.

Reply:
fixed